ably/ably-ui
# Pricing fixes

Jira Ticket Link / Motivation
----



- fix SVG logo looking poorly on Chrome ([and center it again](https://ably.atlassian.net/browse/WEB-2629))
- fix flashes not being removed from the page after hiding (referened here https://github.com/ably/website/pull/5103)
- fix Dropdown component error

Summary of changes
----



 See commits for details.

How do you manually test this?
----



- Logo component should display
- No more error when viewing the Dropdown component
- Flashes close as expected

Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
